examples/Dockerfile.openmpi

```diff
@@ -58,88 +58,75 @@ FROM debian9
 # [1]: https://github.com/open-mpi/ompi/blob/master/README
 
 # OS packages needed to build this stuff.
-RUN apt-get install -y --no-install-suggests \
-    autoconf \
+# As of 10/7/19 CentOS 8 provides verions 22.2 of libibverbs, this is quite new
+# but works for our test systems.
+RUN dnf install -y --setopt=install_weak_deps=false \
+    automake \
     file \
     flex \
-    g++ \
     gcc \
-    gfortran \
+    gcc-c++ \
+    gcc-gfortran \
     git \
-    hwloc-nox \
-    less \
-    libdb5.3-dev \
-    libhwloc-dev \
-    libnl-3-200 \
-    libnl-route-3-200 \
-    libnl-route-3-dev \
-    libnuma1 \
-    libpmi2-0-dev \
+    ibacm \
+    libevent-devel \
+    libtool \
+    libibumad \
+    libibumad-devel \
+    libibverbs \
+    libibverbs-devel \
+    libibverbs-utils \
+    librdmacm \
+    librdmacm-devel \
+    rdma-core \
     make \
-    wget \
-    udev
+    numactl-devel \
+    wget
 
-WORKDIR /usr/local/src
+WORKDIR /usr/src
 
-# Use the Buster versions of libpsm2 (not present in Stretch) and libibverbs
-# (too old in Stretch). Download manually because I'm too lazy to set up
-# package pinning.
-#
-# Note that libpsm2 is x86-64 only:
-#   https://packages.debian.org/buster/libpsm2-2
-#   https://lists.debian.org/debian-hpc/2017/12/msg00015.html
-ENV DEB_URL http://snapshot.debian.org/archive/debian/20181126T030749Z/pool/main
-ENV PSM2_VERSION 11.2.68-3
-RUN if [ "$(dpkg --print-architecture)" = "amd64" ] ; then \
-      wget -nv ${DEB_URL}/libp/libpsm2/libpsm2-2_${PSM2_VERSION}_amd64.deb \
-               ${DEB_URL}/libp/libpsm2/libpsm2-dev_${PSM2_VERSION}_amd64.deb ; \
+# Note that libpsm2 is x86-64 only
+RUN if [ "$(uname -i)" = "x86_64" ] ; then \
+      dnf install -y --enablerepo=PowerTools libpsm2 libpsm2-devel ; \
```

Comment:
What is PowerTools and how trustworthy is it?

Reply:
It's an additional developer repository included in the base image. I can't speak for how reliable it is but I assumed that if it is included in the base image it was probably reliable enough.

According to the [fedora wiki](https://fedoraproject.org/wiki/EPEL)
>EPEL packages assume that the 'PowerTools' repository is enabled. You can do this with: dnf config-manager --set-enabled PowerTools

Reply:
Sounds good.